### PR TITLE
Draft ToC

### PR DESCRIPTION
**Pull request by sunghee-hwang**
Based on Google/Samsung updated proposal and by referring AV1-ISOBMFF specification.

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iac/pull/17.html" title="Last updated on Mar 8, 2022, 5:49 AM UTC (4ed44f5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iac/17/fc99377...4ed44f5.html" title="Last updated on Mar 8, 2022, 5:49 AM UTC (4ed44f5)">Diff</a>

### CODE REVIEW

**Comment by sunghee-hwang** [comment]
When I added RFC6381 reference to index.bs by referring that in AV1-ISOBMFF specification, there was no error during creating index.html on my PC. (curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F force=1 > index.html)
But, when I made it commit to this git, an error created. So, I removed the reference.
Here is errored RFC6381 description:
"RFC6381": {
                        "authors": "R. Gellens;  D. Singer; P. Frojdh",
		"title": "The 'Codecs' and 'Profiles' Parameters for "Bucket" Media Types",
		"status": "August 2011. Proposed Standard",
		"publisher": "IETF",
		"href": "https://tools.ietf.org/html/rfc6381"
	}

File: index.bs [hunk 1]
```diff
@@ -7,9 +7,187 @@ Editor: Felicia Lim, Google, flim@google.com
...
+### Metadata_OBU
+TBA
+
+# Basic Encapsulation Scheme # {#basic-encapsulation}
+This section describes the basic data structures used to signal encapsulation of IA bitstreams in [[!ISOBMFF]] containers.
+
+## General Requirements & Brands ## {#brands}
+A file conformant to this specification satisfies the following:
+- It SHALL conform to the normative requirements of [[!ISOBMFF]]
+- It SHALL have the <dfn value export for="ISOBMFF Brand">aiac(tentative)</dfn> brand among the compatible brands array of the FileTypeBox
```
Comment (tdaede) [inline index.bs:120]: You probably don't need to say (tentative) here as the spec already has a banner on it.

File: index.bs [hunk 2]
```diff
@@ -7,9 +7,187 @@ Editor: Felicia Lim, Google, flim@google.com
...
+TBA
+
+## IA Codec Configuration Box ## {#iacodecconfigurationbox-section}
+This section describes a specific codec configuration box for the codec which is used for the coded audio data of an IA bitstream.
+
+### OPUS Codec Configuration Box ### {#opuscodecconfigurationbox}
+
+#### Definition #### {#opuscodecconfigurationbox-definition}
+<pre class="def">
+	Box Type:  <dfn export>dOps (or opus)</dfn>
```
Comment (tdaede) [inline index.bs:153]: Should be just dOps as that is the only allowed box.

File: index.bs (hunk 1 above)
Comment (sunghee-hwang) [reply index.bs:120]: You are right. I will fix it. Anyway SG need to confirm the codingname.

File: index.bs [hunk 3]
```diff
@@ -7,9 +7,187 @@ Editor: Felicia Lim, Google, flim@google.com
...
+		"href": "http://standards.iso.org/ittf/PubliclyAvailableStandard/c068960_ISO_IEC_14496-12_2015.zip"
+	},
+	"OPUS-ISOBMFF": {
+		"title": "Encapsulation of Opus in ISO Base Media File Format",
+		"status": "April 2016. International Standard",
+		"publisher": "Xiph",
+		"href": "https://opus-codec.org/docs/opus_in_isobmff.html"
+	},
+	"RFC2119": {
+                        "authors": "S. Bradner",
```
[the remark this answers is not in the document]
Comment (sunghee-hwang) [reply index.bs]: fixed them.

File: index.bs (hunk 1 above)
Comment (sunghee-hwang) [reply index.bs]: fixed it.

File: index.bs (hunk 2 above)
Comment (tdaede) [reply index.bs:153]: In general yes, but this is the Opus specific section so dOps is the only one that makes sense, I think?

**Comment by tdaede** [comment]
The issue is the quotes in the string, they need to be escaped:

```
"RFC6381": {
"authors": "R. Gellens; D. Singer; P. Frojdh",
"title": "The 'Codecs' and 'Profiles' Parameters for \"Bucket\" Media Types",
"status": "August 2011. Proposed Standard",
"publisher": "IETF",
"href": "https://tools.ietf.org/html/rfc6381"
}
```

File: index.bs (hunk 2 above)
Comment (sunghee-hwang) [reply index.bs:153]: Understood. It seems to me that the word "codecconfigurationbox" makes a confusion. Here 'opus' box is not audiosampleentry but just a child box which has the same syntax with opusaudiosampleentry. In this case, my intention was opuscodecconfigurationbox = opusaudiosampleentrybox.

**Comment by sunghee-hwang** [comment]
> The issue is the quotes in the string, they need to be escaped:
> 
> ```
> "RFC6381": {
> "authors": "R. Gellens; D. Singer; P. Frojdh",
> "title": "The 'Codecs' and 'Profiles' Parameters for \"Bucket\" Media Types",
> "status": "August 2011. Proposed Standard",
> "publisher": "IETF",
> "href": "https://tools.ietf.org/html/rfc6381"
> }
> ```

Thanks Thomas!

**Comment by sunghee-hwang** [comment]
I would ask you to confirm this even thought it only contains ToC level information.

**Pull request merged**